lms/djangoapps/teams/static/teams/js/views/teams.js

```diff
@@ -0,0 +1,26 @@
+;(function (define) {
+    'use strict';
+    define([
+        'teams/js/views/team_card',
+        'common/js/components/views/paginated_view'
+    ], function (TeamCardView, PaginatedView) {
+        var TeamsView = PaginatedView.extend({
+            type: 'teams',
+            cardView: TeamCardView,
+
+            events: {
+                'click button.action': '' // entry point for team creation
+            },
+
+            render: function () {
+                PaginatedView.prototype.render.call(this);
+
+                this.$el.append(
+                    $('<button class="action action-primary">' + gettext('Create new team') + '</button>')
```

Comment:
@catong I just spoke with @frrrances and @explorerleslie, who suggested that we actually de-emphasize this button, since we only want users to create teams as a last resort. We need copy which prompts the user to do these things, in priority order.

E.g. "Can't find a team? Try [viewing all topics](#), [searching](#), or [creating a new team](#)."